src/plugins/securestorage.ts

```diff
@@ -47,14 +52,16 @@ export class SecureStorage {
   private _objectInstance: any;
   init: Promise<any>;
 
+  constructor()  {}
+
   /**
    * Creates a namespaced storage. 
-   * @param storage {string}
+   * @param store {string}
```

Comment:
I think it's better to set this param name to `name` instead of `store`, don't you think?